docs/conf.py

```diff
@@ -160,6 +160,7 @@ def reset_mpl(gallery_conf, fname):
         "pyrubberband": "https://pyrubberband.readthedocs.io/en/stable/",
         "samplerate": "https://python-samplerate.readthedocs.io/en/latest/",
         "pooch": "https://www.fatiando.org/pooch/latest/",
+        "soxr": "https://github.com/dofuuz/python-soxr",
```

Comment:
I don't think this works?  These links are for intersphinx / documentation cross-referencing.  They will only work on a sphinx-generated documentation site, not a github repo.

Reply:
OK, I'll remove this